style.css

```diff
@@ -67,3 +67,274 @@ main {
 .icons {
   padding: 20px 20px 0 0;
 }
+
+.img-1 {
+  border-radius: 8px;
+}
+
+.canopy-backend ul {
+  display: flex;
+  align-items: center;
+}
+
+.canopy {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  width: 55px;
+  height: 16px;
+  color: #344563;
+}
+
+.counter {
+  margin-left: 10px;
+}
+
+.back-end {
+  margin-left: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  width: 88px;
+  height: 16px;
+}
+
+.date {
+  margin-left: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.daily-read {
+  width: 295px;
+  height: 76px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.dev-lang ul {
+  display: flex;
+  flex-direction: row;
+}
+
+.lang {
+  margin-left: 5px;
+  background: #ebebff;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  height: 24px;
+  padding: 4px 12px;
+  align-items: center;
+  color: #6070ff;
+}
+
+.see-pr {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 16px;
+  margin-top: 20px;
+  margin-left: 10px;
+  padding: 16px;
+  width: 35%;
+}
+
+.card-one {
+  margin-left: 20px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: #fff;
+  padding: 16px;
+}
+
+.card-two {
+  margin-left: 20px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: #fff;
+  padding: 16px;
+}
+
+.card-three {
+  margin-left: 20px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: #fff;
+  padding: 16px;
+}
+
+.card-four {
+  margin-left: 20px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: #fff;
+  padding: 16px;
+}
+
```

Comment:
Please instead of repeating yourself several times while using the same styles, kindly give all cards one class name and give it one style to make your styles `DRY` (**Don't Repeat Yourself)**